Mathlib/Data/Nat/Prime.lean

```diff
@@ -780,6 +781,8 @@ theorem coe_nat_inj (p q : Nat.Primes) : (p : ℕ) = (q : ℕ) ↔ p = q :=
   Subtype.ext_iff.symm
 #align nat.primes.coe_nat_inj Nat.Primes.coe_nat_inj
 
+instance countable : Countable Primes := ⟨⟨coeNat.coe, coe_nat_injective⟩⟩
```

Comment:
I think it would be better for this instance to be beside the `Infinite Primes` instances.